backend/app/Providers/RepositoryServiceProvider.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
You forget to include your Service Provider into your app
Need to add it into config/app.php